### ISSUE TITLE
Make performance counters locally

### ISSUE THREAD

**Issue by msimberg** [STEllAR-GROUP/hpx#4568]

**Comment by hkaiser** [comment]
This might be difficult to achieve or might require significant effort. Currently all performance counters are components that are registered with the symbolic namespace in AGAS. In order to make them available with the local-only runtime we will have to make them non-components and create a new registry for instantiated performance counters. There might be more....

**Comment by msimberg** [comment]
Yeah, this is me optimistically just documenting what could be done. I imagined that it may be possible to wrap a local-only performance counter to turn it into a distributed one (with support e.g. for the locality syntax) but this is without having looked at the implementation at all. This may indeed be difficult.
